### PR TITLE
Update kubectl version to 1.20.6-2 fix security vulnerability

### PR DESCRIPTION
**Pull request by bhatpmk**
# Description

This change updates the kubectl version in the Dockerfile of platform operator and analysis tool to v..20.6-2.

# Checklist 

As the author of this PR, I have:

- [ ] Checked that I included or updated copyright and license notices in all files that I altered
- [ ] Added or updated unit tests for any new functions I added
- [ ] Added or updated integration tests if appropriate
- [ ] Added or updated acceptance tests if appropriate

Code reviewer, please confirm this PR:

- [ ] Addressed the requirement and meets the acceptance criteria
- [ ] Does not introduce unrelated or spurious changes
- [ ] Does not introduce any unapproved dependency
- [ ] Makes sense and it easy to understand, and/or difficult areas of code are clearly documented so that they can be understood

### CODE REVIEW

File: platform-operator/Dockerfile [hunk 1]
```diff
@@ -22,12 +22,12 @@ FROM ghcr.io/oracle/oraclelinux:7-slim
 ARG VERRAZZANO_APPLICATION_OPERATOR_IMAGE
 
 # Use both olcne11 and olcne12 versions since helm-3.1.1 is available only on olcne11 and
```
Comment (bobdonat) [inline platform-operator/Dockerfile:24]: Looks like the comment is inaccurate

Comment (bhatpmk) [reply platform-operator/Dockerfile]: Fixed now, thank you @bobdonat

**Review by markxnelson — DISMISSED** [review]
i need to see this pass in all of the tests before we merge it please

File: platform-operator/Dockerfile (hunk 1 above)
Comment (bobdonat) [reply platform-operator/Dockerfile]: Comment is still not accurate.  The comment is referencing helm-3.1.1 which is not what is being installed.  Also, is olcne11 really still needed.

**Review by bobdonat — APPROVED** [review]

File: platform-operator/Dockerfile (hunk 1 above)
Comment (bhatpmk) [reply platform-operator/Dockerfile]: @bobdonat I have removed olcne11 now.

**Comment by bhatpmk** [comment]
> i need to see this pass in all of the tests before we merge it please

@markxnelson All the tests passed with the change, except a suite which is known to fail intermittently and passed on re-run.

**Pull request merged**